gcp_airflow_foundations/base_class/template_ingestion_options_config.py

```diff
@@ -0,0 +1,23 @@
+from dacite import Config
+from pydantic import validator, root_validator
+from pydantic.dataclasses import dataclass
+
+from typing import List, Optional
+import regex as re
+
+from gcp_airflow_foundations.enums.schema_source_type import SchemaSourceType
+
+
+@dataclass
+class TemplateIngestionOptionsConfig:
+    """
+    Attributes:
+        ingest_all_tables: if true, ingest all tables from source
```

Comment:
If False use regular expressions?

Reply:
Should we add validations for this? 
Also, wouldn't regex_pattern="*" do the same?

Reply:
There is validation in the SourceTemplateConfig class for providing regex_pattern and ingest_all_tables